src/categorical_algebra/CSets.jl

```diff
@@ -352,4 +356,29 @@ function (::Type{T})(X::ACSet, FOb::AbstractDict,
   migrate!(Y, X, FOb, FHom)
 end
 
+""" Serialize an ACSet object to a JSON string
+"""
+serialize(x::ACSet) = JSON.json(x.tables)
+
+""" Deserialize a dictionary from a parsed JSON string to an object of the given ACSet type
+"""
+function deserialize(input::Dict, type)
+    out = type()
```

Comment:
All code in Catlab should have indent width 2.

Reply:
fixed